docs/setup/paths_template.m

```diff
@@ -18,8 +18,6 @@
   rig = thishost;
 end
 
-server1Name = '\\zserver.cortexlab.net';
```

Comment:
The idea behind these is that it is generalizable to other labs and makes it so that if these remote servers ever change, you only have to change the name in one place (where the variables are defined). If you change `server1Name` to znas and `server2Name` to zserver this should be good

Reply:
I agree